tests/test_metadata_cli.py

```diff
@@ -0,0 +1,66 @@
+import responses
+
+from cap_client.utils import json_dumps
+
+
+@responses.activate
+def test_metadata_get(cli_run):
+    responses.add(
+        responses.GET,
+        'https://analysispreservation-dev.cern.ch/api/deposits/some-pid',
+        json={'metadata': {
+            "general_title": "test"
+        }},
+        status=200)
+
+    res = cli_run("metadata get -p some-pid")
+
+    assert res.exit_code == 0
+    assert res.stripped_output == json_dumps({"general_title": "test"})
+
+
+# @responses.activate
+# def test_metadata_update_with_field(cli_run):
```

Comment:
you can use @mark.skip or @mark.fail with the description message (like `should be fixed`) for cases like this
you can read [here](http://doc.pytest.org/en/latest/skipping.html) a bit:)